backend/src/models/question.js

```diff
@@ -35,6 +35,7 @@ class Question {
     postQuestion(courseID, { userID, title, body }) {
         return this.db
             .insert('question', { courseID, userID, title, body })
+            .then(() => this.getQuestions(courseID))
```

Comment:
Still don't think we should do it like this. :(. But it's increasingly baked into the codebase.